sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3367,7 +3409,8 @@ object GpuOverrides extends Logging {
       ExecChecks(
         (TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64 +
           TypeSig.MAP + TypeSig.ARRAY + TypeSig.STRUCT)
-            .nested()
```

Comment:
This change feels like we have regressed.  And empty nested means everything in the original type. The new code drops Maps from what is allowed as a nested value.